src/digraphs.c

```diff
@@ -25,6 +25,14 @@
 static Obj FuncDIGRAPH_OUT_NBS(Obj self, Obj digraph, Obj source, Obj range);
 static Obj FuncDIGRAPH_SOURCE_RANGE(Obj self, Obj digraph);
 
+#if !defined(GAP_KERNEL_MAJOR_VERSION) || GAP_KERNEL_MAJOR_VERSION < 3
+// compatibility with GAP <= 4.9
+static inline Obj NEW_PLIST_IMM(UInt type, Int plen)
+{
```

Comment:
Brace at the end of the previous line please @fingolfin

Reply:
Oops, indeed! Will fix and rebase on stable-0.13